FRW/Modules/Explore/ExploreTabScreen.swift

```diff
@@ -287,6 +286,17 @@ extension ExploreTabScreen {
                 .cornerRadius(28)
         }
     }
+
+    func iconUrl(urlStr: String) -> URL? {
+        guard let url = URL(string: urlStr), let host = url.host() else {
+            return urlStr.toFavIcon()
```

Comment:
Let's update this func `toFavIcon`.
https://discord.com/channels/1166611770404782101/1371269463533682688/1371498729080623135